### ISSUE TITLE
emacs appears to like the line number prior to the path of the file

### ISSUE THREAD

**Issue by snits** [vrothberg/vgrep#96]
Currently vgrep --show will just open up the file with the cursor located at the very beginning of the file. It works with the following quick hack:

```
diff --git a/vgrep.go b/vgrep.go
index 71e853991dbb..40c36bdf8eab 100644
--- a/vgrep.go
+++ b/vgrep.go
@@ -827,6 +827,9 @@ func (v *vgrep) commandShow(index int) bool {

        logrus.Debugf("opening index %d via: %s %s %s", index, editor, path, lFlag)
        cmd := exec.Command(editor, path, lFlag)
+       if strings.Contains(editor, "emacs") {
+               cmd = exec.Command(editor, lFlag, path)
+       }
        cmd.Stdin = os.Stdin
        cmd.Stdout = os.Stdout
        cmd.Stderr = os.Stderr
```

**Comment by vrothberg** [comment]
Oh dear, and I naively mentioned emacs in the readme - should have tested it. Thanks a lot for the report!

**Comment by vrothberg** [comment]
Fixed with #97
